cmd/lotus-miner/init.go

```diff
@@ -214,6 +217,19 @@ var initCmd = &cli.Command{
 				return err
 			}
 
+			if len(build.BuiltinActorsV8Bundle()) > 0 {
+				bs := blockstore.NewMemory()
+
+				if err := actors.LoadBundle(context.TODO(), bs, actors.Version8, build.BuiltinActorsV8Bundle()); err != nil {
+					return xerrors.Errorf("error loading actor bundle: %w", err)
+				}
+
+				cborStore := cbor.NewCborStore(bs)
+				if err := actors.LoadManifests(ctx, cborStore); err != nil {
+					return xerrors.Errorf("error loading actor manifests: %w", err)
+				}
+			}
```

Comment:
Can we put this as a function in chain/actors? Something like 'LoadManifestsFromBuiltin'? We'll probably need to call this in a few more places

Reply:
sure!

Reply:
the other place where we could call it we also have logic for v7 loading, so it doesn't quite map; but this is done, added `actors.LoadManifestFromBundle` which combines `LoadBundle` and `LoadManifests`.

Reply:
see https://github.com/filecoin-project/lotus/pull/8429/commits/beb3369c765e0d813b01fbc71243693380a892fc